BlueprintUI/Sources/Layout/Opacity.swift

```diff
@@ -0,0 +1,60 @@
+import UIKit
+
+/// Changes the opacity of the wrapped element.
+public struct Opacity: Element {
+
+    /// The content element whose opacity is being affected.
+    public var wrappedElement: Element
+
+    /// The opacity of the wrapped element.
+    public var opacity: CGFloat
+
+    /// Initializes an `Opacity` with the given content element and opacity.
+    ///
+    /// - parameters:
+    ///   - opacity: The opacity to be applied to the wrapped element.
+    ///   - wrapping: The content element to be made transparent.
+    public init(
+        opacity: CGFloat,
```

Comment:
Was going to ask if we need to `allowsGroupOpacity`, but apparently I am 7 years out of date and this is now the default! https://developer.apple.com/documentation/quartzcore/calayer/1621277-allowsgroupopacity

Reply:
I've literally never heard of that property 🙈